config/crd/bases/mutations.gatekeeper.sh_assigns.yaml

```diff
@@ -0,0 +1,190 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.3.0
+  creationTimestamp: null
+  name: assigns.mutations.gatekeeper.sh
+spec:
+  group: mutations.gatekeeper.sh
+  names:
+    kind: Assign
+    listKind: AssignList
+    plural: assigns
```

Comment:
Should the plural just be "assign"?

I'm a bit biased here as I really don't like K8s use of plurals, I think having multiple names causes more problems than it solves.

In any case whatever we choose should be consistent with `assignmetadata`

Reply:
Changed plurals of `assign` and `assignmetadata`  to be consistent (singular names).

Fun fact: controller-gen (flect) perceives `assignmetadata` plural as `assignmetadata`  probably due to the `ata` suffix

Reply:
Interesting. Programatically managing these grammar details seems like a hard thing to do.